File: packages/nx-boot-gradle/src/generators/application/files/src/main/java/__packageDirectory__/__appClassName__.java
package <%= packageName %>;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "<%= groupId %>")
public class <%= appClassName %> {

  public static void main(String[] args) {
    SpringApplication.run(<%= appClassName %>.class, args);
  }

}

File: packages/nx-boot-gradle/src/generators/application/files/kotlin/src/main/kotlin/__packageDirectory__/__appClassName__.kt
package <%= packageName %>

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = arrayOf("<%= groupId %>"))
class <%= appClassName %>

fun main(args: Array<String>) {
  runApplication<<%= appClassName %>>(*args)
}


